include/psp2/registrymgr.h

```diff
@@ -82,6 +82,103 @@ int sceRegMgrSetKeyInt(const char* category, const char* name, int buf);
  */
 int sceRegMgrSetKeyStr(const char* category, const char* name, char* buf, const int size);
 
+/**
+ * Get all key's initial information by category (from os0:kd/registry.db0)
```

Comment:
`key's` to ``keys'` and `initial information` to `default values`.

Reply:
good idea